resources/js/views/Teams/Teams.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <section>
+        <div class="columns">
+
+            <div class="column is-one-fifth teams-left-col">
+                <p class="teams-title">OpenLitterMap Teams</p>
+
+                <div v-for="i in items" class="team-flex" @click="goto(i.component)">
```

Comment:
Should use a better name instead of i.

Reply:
We will. This is 1st draft to get something working quickly